### DISCUSSION TITLE
How can I specify a one-way in the lane Quest?

### DISCUSSION THREAD

**Discussion by mcliquid** [streetcomplete/StreetComplete#3985] [Q&A] [answered]
I am currently traveling in a city which is very less mapped. All streets are mapped as residential, but so far without any attributes. Most of these streets are actually one-way streets. In the lane quest, however, I am only asked whether there are two or more lanes.
How can I change the street to a one-way street in the lane quest? Thanks for your help!

**Comment by westnordost** [comment, marked as answer]
This is currently not possible. However, if that road has no lane markings, then you can answer that it has no lane markings. If it has no lane markings, next you will be asked about its width. If the width is below 4m (IIRC), StreetComplete will ask whether it is a oneway.
